Sources/SwiftGen/Config/Config.swift

```diff
@@ -57,11 +57,13 @@ extension Config {
     guard let config = yaml as? [String: Any] else {
       throw Config.Error.wrongType(key: nil, expected: "Dictionary", got: type(of: yaml))
     }
-    self.inputDir = (config[Keys.inputDir] as? String).map { Path($0) }
-    self.outputDir = (config[Keys.outputDir] as? String).map { Path($0) }
+    self.inputDir = (config[Keys.inputDir.rawValue] as? String).map { Path($0) }
+    self.outputDir = (config[Keys.outputDir.rawValue] as? String).map { Path($0) }
+
     var cmds: [String: [ConfigEntry]] = [:]
```

Comment:
Just realised: it might not be useful to parse the Config commands as a `[String: [ConfigEntry]]` anymore if it's only to later transform that representation into a `[(String, ConfigEntry)]` or `[(Parser, ConfigEntry)]` list in `collectCommandsAndEntries()`

Do we use the dictionary (one parser -> list of entries for that parser) structuration in any way other than here? If not, we might as well generate the flatten list right away and get rid of `collectCommandsAndEntries` converting it later 😉